sql/moz-fx-data-shared-prod/glam_etl/glam_desktop_beta_aggregates/view.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-glam-prod-fca7.glam_etl.glam_desktop_beta_aggregates`
```

Comment:
I guess these should move into a `sql/moz-fx-data-glam-prod-fca7` folder

Reply:
Done!